jraph/_src/utils_test.py

```diff
@@ -167,6 +167,69 @@ def _get_list_and_batched_graph():
   return list_graphs, batched_graph
 
 
+def _get_list_matrix():
+  """Returns a list of adjacency matrices, its sparse version
```

Comment:
could you make this comment fit on one line?

Returns sparse and graphstuple version of adj matrix.